src/Cli/Commands/ConfigureOptions.cs

```diff
@@ -79,6 +93,21 @@ public ConfigureOptions(
         [Option("runtime.graphql.multiple-mutations.create.enabled", Required = false, HelpText = "Enable/Disable multiple-mutation create operations on DAB's generated GraphQL schema. Default: true (boolean).")]
         public bool? RuntimeGraphQLMultipleMutationsCreateEnabled { get; }
 
+        [Option("runtime.rest.enabled", Required = false, HelpText = "Enable DAB's Rest endpoint. Default: true (boolean).")]
+        public bool? RuntimeRestEnabled { get; }
+
+        [Option("runtime.rest.path", Required = false, HelpText = "Customize DAB's Rest endpoint path. Allowed values: string. Conditions: Prefix with '/', no spaces and no reserved characters.")]
+        public string? RuntimeRestPath { get; }
+
+        [Option("runtime.rest.request-body-strict", Required = false, HelpText = "Allow/Deny extraneous fields in request body of Rest Schema. Default: true (boolean).")]
+        public bool? RuntimeRestRequestBodyStrict { get; }
+
+        [Option("runtime.cache.enabled", Required = false, HelpText = "Enable DAB's caching of responses globally. Default: false (boolean).")]
```

Comment:
```suggestion
        [Option("runtime.cache.enabled", Required = false, HelpText = "Enable DAB's cache. (You must also enable each entity's cache separately.) Default: false (boolean).")]
```

This is to clarify a confusing behavior of DAB. You must have GlobalCache:On and EntityCache:One for an entity's cache to be enabled.

Reply:
What do we mean by entity cache? Having this flag on would not enable the cache?